File: src/main/java/tri/le/music/util/TokenUtil.java
package tri.le.music.util;

import java.util.UUID;

/**
 * Created by TriLe on 2016-04-02.
 */
public class TokenUtil {

    private static final int TOKEN_SIZE = 36;

    public static String generateToke() {
        return UUID.randomUUID().toString();
    }

    public static boolean validate(String token) {
        if (token != null && token.length() == TOKEN_SIZE)
            return true;

        return false;
    }
}

File: src/main/java/tri/le/music/util/TokenUtil.kt
@file:JvmName("TokenUtil")

package tri.le.music.util

import org.apache.logging.log4j.LogManager
import java.util.*

/**
 * Created by TriLe on 2016-04-22.
 */
private val log = LogManager.getLogger("TokenUtil")
private val TOKEN_SIZE = 36

fun generateToke() = UUID.randomUUID().toString()

fun validate(token: String) = token.length == TOKEN_SIZE
